vta/apps/tsim_example/cmake/modules/hw.cmake

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 if(MSVC)
-  message(STATUS "TSIM build is skipped in Windows..")
+  message(STATUS "[HW] build is skipped in Windows..")
```

Comment:
Consider change to [TSIM_HW]? avoid name conflict

Reply:
Good catch, I just did.